rules/autoconsent/cookie-law-info.json

```diff
@@ -5,7 +5,11 @@
   "detectPopup": [{ "visible": "#cookie-law-info-bar" }],
   "optIn": [{ "click": "[data-cli_action=\"accept_all\"]" }],
   "optOut": [
-    { "hide": ["#cookie-law-info-bar"] },
+    { "click": "#cookie-law-info-bar #wt-cli-settings-btn"},
+    {
+      "wait": 100
```

Comment:
Relying on a timeout may not work for some users. Could we use `waitForThenClick` action instead?